chapters/en/slides/module7/module7_02.Rmd

```diff
@@ -0,0 +1,342 @@
+rather than dragging with the mouse.
+
+In this slide we use `selection_multi`
+to create a selection based on what we click with the mouse.
+If we hold the shift key,
+we can select multiple bars
+(to disable this use `selection_single` instead).
+
+You can also see that instead of changing the color of the bar on selection,
+we here use `alt.condition` to change the opacity instead.
+Whether we change the color or opacity on selection is mostly an aesthetic choice.
+
+We assign the chart to a variable name because we will reuse it in the next slide
+and set the dimensions of the plot to fit on later slides 
+with more complex layouts.
+
+---
+
+## Combining charts with different selections
+
+```{python}
+brush = alt.selection_interval()
+points = (alt.Chart(cars).mark_circle().encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.condition(brush, 'Origin', alt.value('lightgray')))
+.add_selection(brush)).properties(width=300, height=200)
+
+points & bars
+```
+
+Notes:
+To add two charts together,
+we can use what we learned previously about Altair's layout operators.
+To concatenate the charts vertically,
+we're here using the `&` operator.
+
+In contrast to the two scatterplots in an earlier slide,
+these charts are not created from the same base chart,
+so there selections are not linked.
+
+What if we want to link the selections
+so that when we select a region of origin in the bar chart
+the points in the scatter plot that are from the same origin would be highlighted?
+
+This would be useful to study observations from each Origin separately,
+so how can we perform this linkage in Altair?
+
+---
+
+## Linking selections from separate charts requires explicit columns to select on
+
+```{python}
+click = alt.selection_multi(fields=['Origin'])
+bars = bars.add_selection(click)
+points = points.encode(opacity=alt.condition(click, alt.value(0.9), alt.value(0.2)))
+
+points & bars
+```
+
+Notes:
+To link separate charts together,
+we need to be explicit about what happens when a selection is made.
+
+For example,
+to link a click of a bar 
+to a selection in the scatterplot,
+we need to say which column in the dataframe should be used to select on.
+
+In this case,
+we only have a single column that we use in the chart `"Origin"`,
+so this is both our only option
+and what we want since both the barchart and scatterplot
+contain this column as an encoding in the chart.
+
+To successfully link this selection
+we need to do three things:
+
+1. Create a new selection that specifies which columns should be used for selections (the `fields` parameter).
+2. Add this new selection to the bar plot
+3. Add an encoding to the scatterplot that depends on the new `click` selections. Here we use `opacity`.
+
+Now that we click the bars
+the opacity of the corresponding points changes
+as we would expect!
+
+Note that we can still click and drag in the scatterplot
+since our `brush` selection is unchanged from previously.
+
```

Comment:
We show this with transform filter a few slides later. The order could be switch to show that first but I think it is also natural to talk about how to link clicks to a legend instead of a chart so keeping as is.